src/archiefvernietigingscomponent/destruction/models.py

```diff
@@ -102,11 +103,12 @@ def next_assignee(self, review=None):
         #  all reviews have approve status -> list is about to be completed
         return None
 
-    def assign(self, assignee):
-        self.assignee = assignee
-        is_reviewer = assignee != self.author
+    def assign(self, assigned_user: User) -> None:
```

Comment:
Please set the `assigned_on` here (note the point about  having to retrieve the assignee first from the previous comment). Otherwise, If you do it in the forms and the  first reviewer waits 7 days before reviewing, the second reviewer will get a reminder as soon as they are assigned to the list.